modules/economy.js

```diff
@@ -1,295 +1,255 @@
+          }
+        })
       } else {
-        module.exports.getDailyMoney(client, fromUser, guild, false)
-        module.exports.makeMoneyTransferToUser(client, guild, fromUser, toUser, amount, callback)
+        throw new Error('ECO_XX01')
       }
     })
   },
   fetchShopProducts: (client, guild, callback) => {
-    client.pool.query('SELECT * FROM `guildEconomyProducts` WHERE guild = ?', [guild.id], (err, rows) => {
+    client.pool.query('SELECT * FROM `guildEconomyProducts` WHERE guild = ?', [guild.id], (err, shopProducts) => {
       if (err) client.logError(err)
-      if (rows && Object.prototype.hasOwnProperty.call(rows, 0)) {
-        callback(rows)
+      if (shopProducts && Object.prototype.hasOwnProperty.call(shopProducts, 0)) {
+        callback(shopProducts)
       } else {
-        callback(null)
+        throw new Error('ECO_XX02')
       }
     })
   },
-  fetchShopProduct: (client, guild, product, callback) => {
-    client.pool.query('SELECT * FROM `guildEconomyProducts` WHERE guild = ? AND productName = ?', [guild.id, product], (err, rows) => {
+  getShopProduct: (client, guild, productname, callback) => {
+    client.pool.query('SELECT * FROM `guildEconomyProducts` WHERE guild = ? AND productName = ? OR productId = ? LIMIT 1', [guild.id, productname, productname], (err, rows) => {
       if (err) client.logError(err)
       if (rows && Object.prototype.hasOwnProperty.call(rows, 0)) {
-        callback(rows[0])
+        if (callback) callback(rows[0])
       } else {
-        client.pool.query('SELECT * FROM `guildEconomyProducts` WHERE guild = ? AND productId = ?', [guild.id, product], (err, rows) => {
-          if (err) client.logError(err)
-          if (rows && Object.prototype.hasOwnProperty.call(rows, 0)) {
-            callback(rows[0])
-          } else {
-            callback(null)
-          }
-        })
+        if (callback) callback()
       }
     })
   },
-  buyItem: (client, member, guild, productData, callback) => {
-    module.exports.fetchUserAccount(client, member, guild, (userAccount) => {
-      const status = []
-      if (parseInt(productData.productPrice) <= parseInt(userAccount.amount)) {
-        module.exports.itemActivate(client, member, guild, productData, (iaData, continueBuying) => {
-          status.ia = iaData
-          if (continueBuying) module.exports.updateUserAccount(client, member, guild, parseInt(userAccount.amount) - parseInt(productData.productPrice), () => { })
-          if (continueBuying) status.code = true
-          status.code = status.code || false
-          callback(status)
-        })
-      } else {
-        callback(status.code || false)
-      }
+  updateMemberBalance: (client, member, guild, newBalance, callback) => {
+    // TODO: Update column name from "amount" to "balance"
+    client.pool.query('UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?', [newBalance, member.id, guild.id], (err) => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/alexandrumuntenas/pingu/pullRequest?prid=8657799)
```suggestion
    client.pool.query("UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?", [newBalance, member.id, guild.id], (err) => {
```